.github/workflows/scripts/pre-publish-hook

```diff
@@ -13,50 +13,6 @@ echo updating $f...
 condition="'\$(GITHUB_ACTIONS)' == 'true'"
 yq -i -o xml -p xml '
   ( del( .Project.ItemGroup[0] | select(.+@Include == "..\docs\**\*") )
-  | .Project.PropertyGroup = [
-      .Project.PropertyGroup,
-      [
-        {
-          "+@Condition": "'"$condition"'",
-          "ContinuousIntegrationBuild": "true",
-          "EmbedUntrackedSources": "true",
-          "IncludeSymbols": "true",
-          "PublishRepositoryUrl": "true",
-          "SymbolPackageFormat": "snupkg"
-        },
-        {
-          "Copyright": "Copyright (c) Shippo 2024",
-          "Description": ".NET library which integrates with Shippo Multi Carrier Shipping API. This library provides access to Shippo (goshippo.com) API capabilities. Including label generation, rating, tracking and more.",
-          "PackageLicenseFile": "LICENSE",
-          "PackageIcon": "shippo.png",
```

Comment:
This is automatically pulled from icon.png at root according to [docs](https://www.speakeasy.com/docs/publish-sdks#c-nuget)